### ISSUE TITLE
Missing columns parameter in drop method

### ISSUE THREAD

**Issue by fdabek1** [dask/dask#5216]
The current version of Dask is not fully backwards compatible with pandas drop method.  When using the `columns` parameter Dask gives an error: _TypeError: drop() got an unexpected keyword argument 'columns'_. 

Tthe [documentation for drop](https://docs.dask.org/en/latest/dataframe-api.html#dask.dataframe.DataFrame.drop) states that it should work (most likely being pulled from pandas).

Below is a minimum example that should be working, but currently is broken:

```
import dask

df = dask.datasets.timeseries()
df = df.drop(columns=['x', 'y'])
```

Dask Version: 2.2.0

**Comment by martindurant** [comment]
Indeed, the docstrings are copied, with a proviso that there may be differences and, where possible, details of differences. Is this unclear?

In fact, `df.drop(columns=columns)` is identical to `df.drop(columns, axis=1)`, so you could easily work around or indeed implement this in a PR.

**Comment by jrbourbeau** [comment]
Closed via #5223
